passkey/src/lib.rs

```diff
@@ -143,7 +143,7 @@
 //! };
 //!
 //! // Now create the credential.
-//! let my_webauthn_credential = my_client.register(&origin, request, None).await.unwrap();
+//! let my_webauthn_credential = my_client.register(&origin, request, DefaultClientData).await.unwrap();
```

Comment:
**Suggestion:** Mirror the example formatting

```suggestion
//! let my_webauthn_credential = my_client
//!     .register(&origin, request, DefaultClientData)
//!     .await
//!     .unwrap();
```

Reply:
Fixed!